definitions/npm/@koa/router_v8.0.x/flow_v0.84.x-/router_v8.0.x.js

```diff
@@ -0,0 +1,119 @@
+type KoaRouter$Middleware = (
```

Comment:
Any reason to keep those outside of the module definition block?

Reply:
No reason really, as I wrote in the description, I mostly copied a previously existing definition and didn't want to redo everything :)

Reply:
Please tell me if you'd like that I revisit this to merge it.

Reply:
I would personally just declare/export them inside the module scope, without the prefix.

Reply:
Done!